### PR TITLE
style: Add EsLint and Prettier

### PR DESCRIPTION
**Pull request by elsurn**
This PR adds ESLint and Prettier. Also fixes the linting errors and warnings.

### CODE REVIEW

**Comment by col-tom** [comment]
Thanks @elsurn for the work - good job!
I am totally fine with this MR.

What about the empty `.prettierrc.json` - do we need it?

**Comment by elsurn** [comment]
> Thanks @elsurn for the work - good job!
> I am totally fine with this MR.
> 
> What about the empty `.prettierrc.json` - do we need it?

Thanks @col-tom. Yes, it's recommended in the docs to create an empty config file: https://prettier.io/docs/en/install.html

**Comment by col-tom** [comment]
From my side a **GO**
@doublethug What do you think?

File: package.json [hunk 1]
```diff
@@ -48,5 +53,14 @@
...
     "wp": "0.0.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```
Comment (col-tom) [inline package.json:64]: because we primarly work with SASS files, we should includem them as well @elsurn (and mabye exclude css)

Comment (elsurn) [reply package.json:64]: @col-tom good idea! We could use also https://stylelint.io/ . What do you think?

**Review by doublethug — COMMENTED** [review]
Checked out locally and tested, works fine

**Pull request merged**